### PR TITLE
[#1107] Support readme responsive.

### PR DESCRIPTION
**Pull request by tannguyen04**
#[1107](https://github.com/drevops/drevops/issues/1107)

## Checklist before requesting a review

- [x] I have formatted the subject to include ticket number as `[YSCODE-123] Verb in past tense with dot at the end.`
- [x] I have added a link to the issue tracker
- [x] I have provided information in `Changed` section about WHY something was done if this was not a normal implementation
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have run new and existing relevant tests locally with my changes, and they passed
- [x] I have provided screenshots, where applicable

## Changed

1.

## Screenshots

Screenshot 2023-12-26 at 10 15 53 AM

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ Use **tested** Drupal project template with DevOps integrations for CI and hosti
 
 ## Installation
 
-    curl -SsL https://install.drevops.com > install.php && php install.php; rm -r install.php
```
Comment (tannguyen04) [inline README.md:42]: This will create a <pre> tag and it does not work well in td table.

**Comment by AlexSkrypnyk** [comment]
@tannguyen04 
The solution does not allow to copy the code with an on-screen copy code button.

is there any other way? Maybe putting a code block into a table with set width?

**Comment by tannguyen04** [comment]
Hi @AlexSkrypnyk 

I could not find other solution to make pre tag is scrollable inside td. 
But if you allow using DIV tag instead of Table tag, our readme will be fully responsive.

**Comment by AlexSkrypnyk** [comment]
@tannguyen04 please feel free to use any mechanisms available, as long as it renders correctly in the markup and the code has a copy button.

**Comment by AlexSkrypnyk** [comment]
Implemented as inline code block to preserve the table. Closing this.

**Pull request closed without merging**